index.js

```diff
@@ -1,34 +1,24 @@
 document.addEventListener("DOMContentLoaded", () => {
   const addListButton = document.getElementById('submit-list')
   const taskFormDiv = document.getElementById('task-div')
-  let titleOfTheList
```

Comment:
We can use "let" inside since it is only block variable scope, I also adjusted other files for it.